src/style.css

```diff
@@ -1 +1,32 @@
-/* add styles here */
+body {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars:first-child {
+  margin-top: 0;
```

Comment:
Avoid setting margins on specific elements as a selector like `.stars:first-child`. It's better to use a class that can be applied to any element needing this style, ensuring consistency and preventing unexpected behaviors. For example, you could have a utility class like `.mt-0` to set `margin-top: 0;` and apply it to any element needing it.